.github/workflows/test.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       - run: npm run test:integration
         env:
           PUBLIC_KEY: xnd_public_development_2hi8lWQiSiR07H7n9RNt4HDmEGn5hJbUIDejBaqL44XtLzWkLd6ICd4LwJLrXRw
-          SECRET_KEY: xnd_development_chBDpzjQZEI8nncBLrWtwau5r7rgiuunCJ4JCqsd68rXemDd74BnRpdO0bIZAMK
+          SECRET_KEY: xnd_development_rikCabnxWNrM5hNwOA6I81KH8KIcpPX5D7Sp2W4x2assgpsERsAuG9e51R436
```

Comment:
Retail outlet require activation through dashboard.
Change secret key to xendit test account's key with retail outlet already activated.